coalib/settings/ConfigurationGathering.py

```diff
@@ -138,8 +138,12 @@ def load_config_file(filename, log_printer=None, silent=False):
                                           file=filename,
                                           found='does not exist'))
                 sys.exit(2)
-
-        return {'default': Section('default')}
+    except PermissionError:
+        if silent:
```

Comment:
Change it to `if not silent` and so you will have CI failing thus add more test cases ;P

Reply:
@Makman2 what do you suggest?